benchmarks/build-system-cost/main.go

```
@@ -0,0 +1,679 @@
+	}
+	if changed := changedFiles(before, after); len(changed) != 1 || changed[0] != editFile {
+		return armResult{}, fmt.Errorf("edit changed %v, want only %s", changed, editFile)
+	}
+	output.Clean = phases{SetupSeconds: setup, DependencySeconds: dependencies, BuildSeconds: clean, TotalSeconds: setup + dependencies + clean}
+	output.Warm = buildPhase{BuildSeconds: warm}
+	output.Edit = editPhase{EditedFile: editFile, BuildSeconds: edited}
+	return output, nil
+}
+
+func timedCommand(ctx context.Context, directory string, environment []string, logFile *os.File, name string, args ...string) (float64, error) {
+	fmt.Fprintf(logFile, "$ %s %s\n", name, strings.Join(args, " "))
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)